specifyweb/frontend/js_src/lib/components/FormSliders/RecordSet.tsx

```diff
@@ -412,7 +416,33 @@ function RecordSet<SCHEMA extends AnySchema>({
                 })
             : undefined
         }
-        onClone={(newResource): void => go(totalCount, 'new', newResource)}
+        onClone={(resources: RA<SpecifyResource<SCHEMA>>): void => {
+          go(totalCount, 'new', resources[0]);
+          if (resources.length > 1) {
```

Comment:
if a new record set is created, shouldn't you `go()` to the newly created record set?
or does it already do that?

Reply:
it does it yes